modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/SideMenu/index.js

```diff
@@ -84,39 +84,47 @@ const SideMenu = () => {
 			<ul className="list-unstyled mr-2">
 				<MenuItem to="">{MENU_TYPES.overview}</MenuItem>
 
-				<li>
-					<Button
-						appendIcon="angle-right-small"
-						appendIconClassName="ml-auto"
-						className={classNames(
-							'align-items-center btn-borderless d-flex px-3 py-2 rounded text-neutral-10 w-100',
-							{
-								'cp-product-activation-active': isOpenedProductsMenu,
-								'cp-products-list-active': hasSomeMenuItemActive,
+				{subscriptionGroups.length > 0 ? (
```

Comment:
Hey Clarice, i think would be more clean if use length boolean verification instead compare with 0. What do you think?

Other option: create a const more explicit.
Example:
const hasSomeSubscriptionGroup = !!subscriptionGroups.length;

```suggestion
				{!!subscriptionGroups.length ? (
```